specs/deposits.md

```diff
@@ -356,9 +357,10 @@ The version is incremented to `1.2.0` and several new storage slots are used for
 - `blobBasefeeScalar` (`uint256`): The scalar value applied to the L1 blob base fee portion of the L1 cost.
 - `basefeeScalar` (`uint256`): The scalar value applied to the L1 base fee portion of the L1 cost.
 
-Additionally, the `setL1BlockValues` function is deprecated and replaced with `setL1BlockValuesEcotone`.
-`setL1BlockValuesEcotone` uses packed encoding for its parameters, which is described in
-[L1 Attributes Deposited Transaction Calldata](#l1-attributes-deposited-transaction-calldata)
+Additionally, the `setL1BlockValues` function is deprecated and MUST never be called after the Ecotone hardfork
+activation. The `setL1BlockValuesEcotone` MUST be called after the Ecotone hardfork activation.
```

Comment:
>The `setL1BlockValuesEcotone` MUST be called after the Ecotone hardfork activation.

"after" is a bit ambiguous here given that this is consensus critical. Who / what calls this, and when should they call it exactly?

Reply:
Good catch, this should be specific and mention exact block numbers

Reply:
Generally we do this in reference to the ecotone timestamp. So in this case, `setL1BlockValuesEcotone` must be called on the second ecotone block where the first ecotone block is the L2 block whose timestamp == ecotone time.